types/src/transaction/mod.rs

```diff
@@ -980,7 +963,6 @@ impl TransactionStatus {
     pub fn from_vm_status(
         vm_status: VMStatus,
         charge_invariant_violation: bool,
-        features: &Features,
     ) -> (Self, TransactionAuxiliaryData) {
         let status_code = vm_status.status_code();
         let txn_aux = TransactionAuxiliaryData::from_vm_status(&vm_status);
```

Comment:
Do we still need this (`TransactionAuxiliaryData::from_vm_status(&vm_status)` and in general returning `(Self, TransactionAuxiliaryData)`? We are just duplicating the error? I am in favour of removing this population in favour of keeping the code simple.